Documentation/README_Interactable.md

```diff
@@ -226,12 +223,34 @@ public override void SetValue(InteractableThemeProperty property, int index, flo
 ```
 
 The SetValue function is used to set the property value based on the current state.
-**Property**: A list of property values per state, set through the theme inspector.
-**Index**: Correlates to the current state.
-**Percentage**: A float value between 0 and 1. It will be the eased value if an Animation curve is used.
+- **Property**: A list of property values per state, set through the theme inspector.
+- **Index**: Correlates to the current state.
+- **Percentage**: A float value between 0 and 1. It will be the eased value if an Animation curve is used.
 
 Custom Settings added to the new class will also be displayed in the inspector. If there are properties that need to be exposed in the inspector but do not need to be based on states, they should be added to the Custom Settings list.
 
+### Extending Shader Themes
```

Comment:
i would cross link this with the shader article (add a #See Also section on the bottom linking to https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/README_MRTKStandardShader.html?q=shader - and in the standard shader article link to the extending shader themes of this article)